### PR TITLE
Add copy to clipboard for branch/repository name

### LINKED ISSUE #12141: provide feature to copy branch/repository name
### Describe the feature or problem you’d like to solve

If I want to copy the name of the branch or repository that I cloned, no way to do it( than rename -> copy )

### Proposed solution

IT will be easy to copy if the user wants to use it for some other purpose 

### Additional context

Add any other context like screenshots or mockups are helpful, if applicable.

### PR DESCRIPTION
**Pull request by tsvetilian-ty**
Closes #12141

### Screenshots

image
image

### CODE REVIEW

**Review by tidy-dev — CHANGES_REQUESTED** [review]
@tsvetilian-ty  Thank you so much for contributing! I particularly would like this feature. 😀 

However, we have some minor changes we would like for app congruency. 

For the branch menu:

```
Rename...
Copy branch name
-----
Delete...
```

For repo menu:
```
Create alias
Copy repo name
----
Open in
Show in
Open in
----
Remove...
```

**Comment by tsvetilian-ty** [comment]
@tidy-dev the pleasure to contribute to the project is all mine 🙌 
image
image

**Review by tidy-dev — APPROVED** [review]
✨ @tsvetilian-ty Thank you so much for those changes!

File: app/src/ui/branches/branch-list-item.tsx [hunk 1]
```diff
@@ -71,6 +72,13 @@ export class BranchListItem extends React.Component<IBranchListItemProps, {}> {
       })
     }
 
+    items.push({
+      label: __DARWIN__ ? 'Copy Branch Name' : 'Copy branch name',
```
Comment (clary54) [inline app/src/ui/branches/branch-list-item.tsx:76]: clary54

**Pull request merged**